File: 8.10/example2.py
from random import randint

arrayData = []

for y in range(1, 101):
    arrayData += [randint(0, 25565)]


number = 0
count = 0
while number < 10:
    dataInArray = arrayData[count]
    count = count + 1
    if dataInArray >= 100:
        number = number + 1

File: java/8.8/example2.java
import java.util.Random;

public class example2 {
    public static void main(String[] args) {
        Random rand = new Random();
        Integer[] arrayData = new Integer[101];
        for (Integer y = 0; y <= 100; y++) {
            arrayData[y] = rand.nextInt(25565);
        } 
        Integer count = 0;
        for (Integer x = 0; x < 100; x++) {
            if (arrayData[x] > 50) {
                count++;
            }
        }
        System.out.println(count);
    }
}
